test/e2e/app/app.go

```diff
@@ -281,6 +317,10 @@ func (app *Application) CheckTx(_ context.Context, req *abci.RequestCheckTx) (*a
 // It returns a boolean indicating if a blob exists (either retrieved from the simaulted network or from the
 // local cache) and the blob itself.
 func (app *Application) GetBlob(height int64) ([]byte, bool, error) {
+	if !app.checkBlobHeight(height, "getBlob") {
```

Comment:
As this is only called from FinalizeBlock and it's guarded with the check, this is most likely not needed.